### PR TITLE
Es: Dominical upload

### PR DESCRIPTION
**Pull request by Reixaldo**

### CODE REVIEW

File: UI/ui_mainmode.csv [hunk 1]
```diff
@@ -5662,7 +5662,7 @@ Achievement_Down#0,"""を達成しました！"""
 Clear_Msg#0,"""チャレンジクリア率：MISSION$(0) $(1)%"""
 TextStart#0,"""Challenge  start"""
 TextClear#0,"""Challenge clear"""
-TextMission#0,"""mission"""
+TextMission#0,"""Mission"""
```
Comment (alama) [inline UI/ui_mainmode.csv]: you need to keep this to the JP Text `mission` since the EN Text `Mission` will apply in the patches

Comment (Reixaldo) [reply UI/ui_mainmode.csv]: ok, i am sorry.... ><

**Comment by TheBlazter** [comment]
Please keep quiet "dummy", "blank", "ep5 test" lines and tanabata names

File: Misc/Lobby/oa_tanabata.csv [hunk 2]
```diff
@@ -130,21 +130,21 @@ result53#1,"""ブルーノ"""
 result54#1,"""せーわ"""
 result55#1,"""機械仕掛けのさおリリーパ"""
 result56#1,"""クリきんと"""
-result57#1,"""mashiro@uyo"""
+result57#1,"""Mashiro@uyo"""
```
Comment (TheBlazter) [inline Misc/Lobby/oa_tanabata.csv]: Please keep quiet the tanabata names

File: Misc/actor_name.csv [hunk 3]
```diff
@@ -866,93 +866,93 @@ Npc_316#2,"""イツキ（具現武装）"""
 Npc_213#1,"""マトイ（EP4）"""
 Npc_407#2,"""ジェネ(ステライノセント)"""
 Npc_499#2,"""ジェネ(ステラメモリーズ)"""
-Npc_CounterQuest1#2,"""dummy"""
```
Comment (TheBlazter) [inline Misc/actor_name.csv]: Please keep quiet "dummy" lines

File: Misc/actor_name.csv [hunk 4]
```diff
@@ -1560,8 +1560,8 @@ WpnmanSwordPA01#1,"""『大剣』ヒーロー用フォトンアーツ。<br>大
 WpnmanSwordPA02#1,"""『大剣』ヒーロー用フォトンアーツ。<br>回避しながら攻撃を行う。<br>チャージすると、広範囲攻撃になる。"""
 WpnmanSwordPA03#1,"""『大剣』ヒーロー用フォトンアーツ。<br>高速突進の突きを行う。ボタン長押しで<br>導具に持ち替え、追撃する。"""
 WpnmanSwordPA04#1,"""『大剣』ヒーロー用フォトンアーツ。<br>気弾を連射し移動する。ボタン長押しで<br>双機銃に持ち替え、追撃する。"""
-WpnmanSwordPA05#1,"""blank"""
```
Comment (TheBlazter) [inline Misc/actor_name.csv]: Please keep quiet "blank" lines

File: Story/Episode 5/np_014300.csv [hunk 5]
```diff
@@ -5,8 +5,8 @@ text999#0,"""Volver"""
 text01#0,"""テキストがまだないので仮です１"""
 text03#0,"""テキストがまだないので仮です２"""
 text02#0,"""テキストがまだないので仮です３"""
-ep5test01#0,"""ep5スキット1"""
-ep5test02#0,"""ep5スキット2"""
```
Comment (TheBlazter) [inline Story/Episode 5/np_014300.csv]: Please keep quiet the ep5test lines

File: Story/Episode 5/np_014380.csv [hunk 6]
```diff
@@ -3,8 +3,8 @@ text03#0,"""テキストがまだないので仮です２"""
 text02#0,"""テキストがまだないので仮です３"""
 text10#0,"""Comenzar la Batalla con 【Profound Darkness】"""
 text20#0,"""Continuar con los preparativos para la batalla"""
-ep5test01#0,"""ep5スキット1"""
-ep5test02#0,"""ep5スキット2"""
```
Comment (TheBlazter) [inline Story/Episode 5/np_014380.csv]: Please keep quiet the ep5test lines

File: Story/Episode 5/st_054070.csv [hunk 7]
```diff
@@ -7,7 +7,7 @@ st_054070_0055#0,"""Lo siento, soy una persona que camina con alguien rudo y ant
 st_054070_0060#0,"""No, no creo hasta ahora...<br>Aun así, ustedes dos son amigos cercanos.<br>¿Qué tipo de relación tienen?"""
 st_054070_0070#0,"""Bueno, eso es..."""
 st_054070_0080#0,"""Tanto Sina como Yo somos cuerpos experimentales de soldados demoníacos."""
-st_054070_0090#0,"""cuerpo experimental?..."""
+st_054070_0090#0,"""Cuerpo experimental?..."""
```
Comment (TheBlazter) [inline Story/Episode 5/st_054070.csv]: Missing interrogation opening "¿"

File: UI/common.csv [hunk 8]
```diff
@@ -718,7 +718,7 @@ LobbyAction_bubble#0,"""シャボン玉"""
 LobbyAction_enemy2#0,"""エネミー２"""
 LobbyAction_umbrella1#0,"""エア傘"""
 LobbyAction_umbrella2#0,"""リアル傘"""
-LobbyAction_dummy#0,"""dummy"""
+LobbyAction_dummy#0,"""Dummy"""
```
Comment (TheBlazter) [inline UI/common.csv]: Please keep quiet dummy lines

File: UI/ui_item.csv [hunk 9]
```diff
@@ -8571,7 +8571,7 @@ RangeTerm_006#0,"""ホワイトデー"""
 RangeTerm_007#0,"""花見"""
 RangeTerm_008#0,"""季節（春）"""
 RangeTerm_009#0,"""Dia de los inocentes"""
-RangeTerm_010#0,"""esサービスイン"""
+RangeTerm_010#0,"""Esサービスイン"""
```
Comment (TheBlazter) [inline UI/ui_item.csv]: Please keep quiet or translate correctly

File: UI/ui_ship_battle.csv [hunk 10]
```diff
@@ -82,7 +82,7 @@ VeryHard#1,"""Muy Difícil"""
 SuperHard#1,"""Super Difícil"""
 SubPaletteTabChangeTag#0,"""<%TabForward>"""
 SubPaletteTabChange#0,"""Siguiente"""
-ShipName#0,"""ship $(0)"""
+ShipName#0,"""Ship $(0)"""
```
Comment (TheBlazter) [inline UI/ui_ship_battle.csv]: Please translate

**Review by TheBlazter — CHANGES_REQUESTED** [review]

File: Misc/Lobby/oa_tanabata.csv (hunk 2 above)
Comment (Reixaldo) [reply Misc/Lobby/oa_tanabata.csv]: Done

File: Misc/actor_name.csv (hunk 3 above)
Comment (Reixaldo) [reply Misc/actor_name.csv]: Done

File: Misc/actor_name.csv (hunk 4 above)
Comment (Reixaldo) [reply Misc/actor_name.csv]: Done

File: Story/Episode 5/np_014300.csv (hunk 5 above)
Comment (Reixaldo) [reply Story/Episode 5/np_014300.csv]: Done

File: Story/Episode 5/np_014380.csv (hunk 6 above)
Comment (Reixaldo) [reply Story/Episode 5/np_014380.csv]: Done

File: UI/common.csv (hunk 8 above)
Comment (Reixaldo) [reply UI/common.csv]: Done

File: UI/ui_item.csv (hunk 9 above)
Comment (Reixaldo) [reply UI/ui_item.csv]: Done

File: UI/ui_ship_battle.csv (hunk 10 above)
Comment (Reixaldo) [reply UI/ui_ship_battle.csv]: Done

File: Misc/Lobby/oa_tanabata.csv (hunk 2 above)
Comment (alama) [reply Misc/Lobby/oa_tanabata.csv]: No it is not, there still changes in oa_tanabata.csv

File: UI/common.csv (hunk 8 above)
Comment (alama) [reply UI/common.csv]: You also need to undo the changes for "dummy" in quest_common.csv

File: UI/refer.csv [hunk 11]
```diff
@@ -326,8 +326,8 @@ pso2_players_manual_preparations_quest_arksbingocard#0,"""<url http://pso2.jp/pl
 pso2_players_manual_preparations_loginstamp#0,"""<url http://pso2.jp/players/manual/preparations/info/loginstamp/>"""
 pso2_players_manual_preparations_info_arksroad#0,"""<url http://pso2.jp/players/manual/preparations/info/arksroad/>"""
 pso2_players_manual_preparations_party_supportpartner#0,"""<url http://pso2.jp/players/manual/preparations/party/supportpartner/>"""
-AC_CHARGE_PAGE#0,"""https://AC-CHARGE-URL"""
-USER-INFO-CHANGE-URL#0,"""https://USER-INFO-CHANGE-URL"""
+AC_CHARGE_PAGE#0,"""Https://AC-CHARGE-URL"""
```
Comment (alama) [inline UI/refer.csv]: Why are you changing this URL? why all the https:// ?

**Comment by SynthSy** [comment]
the pso2 website does not use https

File: UI/refer.csv (hunk 11 above)
Comment (Reixaldo) [reply UI/refer.csv]: wtf, i fixed that! Dx

File: Misc/Lobby/oa_tanabata.csv [hunk 12]
```diff
@@ -133,18 +133,18 @@ result56#1,"""クリきんと"""
...
-result60#1,"""thebluecat"""
+result60#1,"""Thebluecat"""
 result61#1,"""間取"""
-result62#1,"""ashitubo3"""
+result62#1,"""Ashitubo3"""
 result63#1,"""雛月ユウキ"""
 result64#1,"""kise-yayoi"""
 result65#1,"""Qreuz"""
-result66#1,"""lakota"""
+result66#1,"""Lakota"""
```
Comment (TheBlazter) [inline Misc/Lobby/oa_tanabata.csv]: Still unfixed ( just discard modifications on the file)

**Review by TheBlazter — CHANGES_REQUESTED** [review]

File: UI/common.csv (hunk 8 above)
Comment (Reixaldo) [reply UI/common.csv]: Done

File: UI/ui_mainmode.csv (hunk 1 above)
Comment (Reixaldo) [reply UI/ui_mainmode.csv]: Done

File: Misc/Lobby/oa_tanabata.csv (hunk 2 above)
Comment (Reixaldo) [reply Misc/Lobby/oa_tanabata.csv]: ahm.... done?

File: UI/common.csv (hunk 8 above)
Comment (Reixaldo) [reply UI/common.csv]: Done

File: Misc/Lobby/oa_tanabata.csv (hunk 2 above)
Comment (alama) [reply Misc/Lobby/oa_tanabata.csv]: I still see changes in oa_tanabata.csv
https://github.com/Arks-Layer/PSO2ENPatchCSV/pull/818/files#diff-56acfed6086053c83194edf627c997b0

File: UI/common.csv (hunk 8 above)
Comment (alama) [reply UI/common.csv]: Still changes in quest_common.csv
https://github.com/Arks-Layer/PSO2ENPatchCSV/pull/818/files#diff-6809697f6feaafe95cc372c63532e0b2R404

File: Misc/Lobby/oa_tanabata.csv (hunk 12 above)
Comment (Reixaldo) [reply Misc/Lobby/oa_tanabata.csv]: done

File: Misc/Lobby/oa_tanabata.csv (hunk 2 above)
Comment (alama) [reply Misc/Lobby/oa_tanabata.csv]: and there are STILL CHANGES in oa_tanabata.csv

File: UI/quest_common.csv [hunk 13]
```diff
@@ -3,18 +3,18 @@ f0#0,"""¡Un miembro del grupo ha sido incapacitado!"""
...
-f9#0,"""Prueba fallida! El lider a dejado el Grupo."""
+f9#0,"""¡Prueba fallida! El lider a dejado el grupo."""
 f10#0,"""Todos los sitios han sido infectados!"""
 f11#0,"""A fallado en proteger la base!"""
 f12#0,"""Prueba de emergencia fracasada!"""
 f13#0,"""La misión fue completada por los otros grupos!"""
 f14#0,"""¡Se alcanzo el límite de incapacitación!"""
 f15#0,"""¡Objetivo de defensa destruido!"""
-Test#0,"""Dummy1"""
+Test#0,"""dummy1"""
```
Comment (alama) [inline UI/quest_common.csv]: please undo the dummy changes

File: Misc/Lobby/oa_tanabata.csv [hunk 14]
```diff
@@ -131,19 +131,19 @@ result54#1,"""せーわ"""
 result55#1,"""機械仕掛けのさおリリーパ"""
 result56#1,"""クリきんと"""
 result57#1,"""mashiro@uyo"""
-result58#1,"""Cool _Nicos"""
+result58#1,"""cool _Nicos"""
```
Comment (alama) [inline Misc/Lobby/oa_tanabata.csv]: Undo the changes for this file
This is list of player names, aka, a highscore

Comment (SynthSy) [reply Misc/Lobby/oa_tanabata.csv]: it's why we never bothered translating them

File: UI/quest_common.csv (hunk 13 above)
Comment (Reixaldo) [reply UI/quest_common.csv]: Done!

**Comment by alama** [comment]
> the pso2 website does not use https

but the JP text does
`$ grep -n -R Files -e "pso2_segaid_user_activate_url_switch#0"
Files/refer.csv:358:pso2_segaid_user_activate_url_switch#0,"""https://pso2.jp/ref/SW001/"""
`

File: UI/refer.csv [hunk 15]
```diff
@@ -355,7 +355,7 @@ pso2_players_PVP#0,"""<url http://pso2.jp/players/>"""
 pso2_players_manual_coordinate#0,"""<url http://pso2.jp/players/manual/preparations/equipment/coordinate/>"""
 pso2_players_poritan#0,"""<url http://pso2.jp/players/update/20171108/02/>"""
 pso2_players_manual_preparations_darkblast#0,"""<url http://pso2.jp/players/manual/preparations/darkblast/>"""
-pso2_segaid_user_activate_url_switch#0,"""https://pso2.jp/ref/SW001/"""
+pso2_segaid_user_activate_url_switch#0,"""http://pso2.jp/ref/SW001/"""
```
Comment (alama) [inline UI/refer.csv]: Revert this back to https://

**Review by TheBlazter — APPROVED** [review]
